### ISSUE TITLE
i3wm like layout option

### ISSUE THREAD

**Issue by ashic** [rickbutton/workspacer#174]
In i3wm, I have the following:

1. If I hit Win+V, it goes into vertical mode. Any new app opened is placed below the currently active window.
2. If I hit Win+H, it goes into horizontal mode. Any new app opened is placed to the right of the currently active window. 

Is something similar doable in Workspacer? Currently I see that If I open 4 apps, the first takes the left half of the screen and each subsequent app gets stacked vertically inside the right half of the screen.

**Comment by josteink** [comment]
> Is something similar doable in Workspacer?

Not currently. It's a Workspacer is a tilling WM, but not all tilling WMs are like i3 or sway :smile:

**Comment by rickbutton** [comment]
yeah, workspacer currently follows the "xmonad" style of tiling window manager philosophy, where the WM is 100% responsible for laying out windows for you, given some algorithm (the layout engine). i3/sway are tilers, but let you re-arrange the tiles however you want on the fly, dynamically.

In theory, layout engines can do whatever you want, and they are stateful, so you could implement the entire i3/sway layout strategy in a layout engine (in theory), but it would be a lot of work.

**Comment by josteink** [comment]
Completely untested on my part, but this might be another option more fitting for those who seek a Windows-based i3-equivalent:

https://github.com/McYoloSwagHam/win3wm
